messaging/messaging-core/src/main/java/uk/gov/justice/services/messaging/JsonObjectEnvelopeConverter.java

```diff
@@ -4,13 +4,19 @@
 import javax.json.Json;
 import javax.json.JsonObject;
 import javax.json.JsonObjectBuilder;
+import javax.json.JsonValue;
+import javax.json.JsonValue.ValueType;
+
+import static javax.json.JsonValue.ValueType.ARRAY;
+import static javax.json.JsonValue.ValueType.OBJECT;
 
 /**
  * A converter class to convert between {@link JsonEnvelope} and {@link JsonObject}.
  */
 public class JsonObjectEnvelopeConverter {
 
     public static final String METADATA = "_metadata";
```

Comment:
Does this need to be public? and if yes is it worthwhile having RESULTS public also?